logstash-core/src/main/java/co/elastic/logstash/api/Configuration.java

```diff
@@ -0,0 +1,106 @@
+package co.elastic.logstash.api;
+
+import java.nio.file.Path;
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * LS Configuration example. Should be implemented like Spark config or Hadoop job config classes.
```

Comment:
is it an example, or is it the implementation?

Reply:
It's the implementation. I'll update the javadoc accordingly.